### DISCUSSION TITLE
SoftHSMv2 is back?

### DISCUSSION THREAD

**Discussion by jschlyter** [opendnssec/SoftHSMv2#763] [Announcements]
As some of you may have noticed, we've moved all SoftHSM repositories to https://github.com/softhsm. Most repositories are now archived, but SoftHSMv2 itself is slowly starting to wake up.

If you are willing to help, please reach out to @oej or @jschlyter.

**Comment by invik** [comment]
This is some very good news.
Which type of help is needed ?
As of now, there is a lot of PRs that are outstanding.

**Reply by jschlyter** [reply]
First step is to get the CI up and running to ensure new PRs don't break things. I'm hoping @bjosv and #733 will solve this. Then we can start processing PRs, but we do need help with review.

@oej and myself has also started to discuss financing with some users, we will need funding for long-term maintenance.

**Comment by antoinelochet** [comment]
Hello,
Do you need any help to merge most of the outstanding PRs ?
A lot of those (mine included) have been rebased and are wiating for feedback.

**Reply by jschlyter** [reply]
Code review of pending PRs would be most useful. Also ensure that all PRs are in sync with main.

**Reply by antoinelochet** [reply]
I have done that on mines, I am waiting for core team reviews now.

**Comment by teward** [comment]
Is this new revival of SoftHSMv2 the reason there's been no major software release since 2023?  I'd like to use this to 'emulate' the use of an actual HSM with local development for my software libraries via PKCS11 interfaces, but not if the code is suffering bitrot.

**Reply by jschlyter** [reply]
We're slowing trying to get back on track. Not sure about release plans yet.

**Comment by jschlyter** [comment]
I'm happy to announce that @bukka will join me in managing core reviews and releases. We hope to have a release before the end of the year - the first one in quite a while.

**Reply by jschlyter** [reply]
Also, @antoinelochet will help out with code reviews.

**Reply by teward** [reply]
Nice to see.  Once you guys have a release and something considered 'stable' and not bitrotted, you'll see me testing some things and probably filing bugs/issues stuff back :D
